Doc/howto/argparse.rst

```diff
@@ -788,6 +788,59 @@ but not both at the same time:
      -q, --quiet
 
 
+Internationalization support
+============================
+
+The output of the :mod:`argparse` module such as its help text and error
+messages are all made translatable using the :mod:`gettext` module. This
+allows applications to easily localize messages produced by
+:mod:`argparse`. (If you are not familiar with Internationalization and
+Localization, you can check out :ref:`i18n-howto`).
+
+For instance, in this :mod:`argparse` output:
+
+.. code-block:: shell-session
+
+   $ python prog.py --help
+   usage: prog.py [-h] [-v | -q] x y
+
+   calculate X to the power of Y
+
+   positional arguments:
+     x              the base
+     y              the exponent
+
+   options:
+     -h, --help     show this help message and exit
+     -v, --verbose
+     -q, --quiet
+
+The strings ``usage:``, ``positional arguments:``, ``options:`` and
+``show this help message and exit`` are all translatable.
+
+In order to translate these strings, you will probably want to extract them
+into a ``.po`` file. For example, using `Babel <https://babel.pocoo.org/>`__,
+you can run:
+
+.. code-block:: shell-session
+
+  $ pybabel extract -o messages.po /usr/lib/python3.12/argparse.py
+
+This command will extract all translatable strings from the :mod:`argparse`
+module and output them into a file named ``messages.po``. This command assumes
+that your Python installation is in ``/usr/lib``.
+
+You can find out the location of the :mod:`argparse` module on your system
+using this simple script::
```

Comment:
Avoid _easy_, _simply_, _simply_, etc. What's simple for someone might be challenging and difficult for another.